packages/task/src/node/task-server.ts

```diff
@@ -85,6 +94,11 @@ export class TaskServerImpl implements TaskServer, Disposable {
         if (!this.toDispose.has(task.id)) {
             this.toDispose.set(task.id, new DisposableCollection());
         }
+
+        if (!this.backgroundTaskStatusMap.has(task.id)) {
+            this.backgroundTaskStatusMap.set(task.id, { 'isBeginsPatternMatch': false, 'isEndsPatternMatch': false, 'isActive': false });
+        }
```

Comment:
looks like all tasks are added to the `backgroundTaskStatusMap`, not just background tasks.

i guess we probably want to either renamed the variable `backgroundTaskStatusMap`, or update the if-condition to only allow background tasks to be added.

Reply:
fixed, done